server/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -919,7 +901,7 @@ public boolean forbidClosedIndices() {
     }
 
     /**
-     * @return Whether execution on closed indices is allowed.
+     * @return Whether execution on failure indices is allowed.
```

Comment:
Super minor grammar nit

```suggestion
     * @return Whether execution on failure indices are allowed.
```

Reply:
I reverted it, I just notice that the subject of the verb is execution which is singular.

Reply:
Yes, you are right, sorry about that!